contributing/CODE_OWNERSHIP.md

```diff
@@ -64,13 +66,13 @@ described above. Stay tuned for the latest changes.
 
 ## Approvals required before merging a change
 
-In order to merge a PR, an approval will be required from at least one of the
-owners of each file modified by the PR. See [contributing-code.md](./contributing-code.md)
-for more details.
+Any change in AMP requires the approval of at least one OWNER of the code the change
```

Comment:
Styling nit: I'm not sure where we want to / don't want to capitalize the word "owner". As @tobie mentioned in https://github.com/ampproject/amphtml/pull/21280#pullrequestreview-211662671, we should try to be consistent across our documentation.

Reply:
Good point; done.